components/TradingCardGrid.jsx

```diff
@@ -34,33 +36,26 @@ export default function TradingCardGrid({ teamMembers }) {
   };
 
   useEffect(() => {
-    const filterTeamMembers = (allTeamMembers) => (
-      allTeamMembers.filter((member) => {
-        const hoursIdx = getForecastedHoursIdx(weekOffset);
-        const hasHoursAvailable = member.weeklyCapacity - member.forecastedHours[hoursIdx] > 0;
-
-        if (availability.length !== Object.keys(availabilityOptions).length) {
-          if (
-            !hasHoursAvailable && availability.includes(availabilityOptions.AVAILABLE)
-          ) return false;
-          if (
-            hasHoursAvailable && availability.includes(availabilityOptions.UNAVAILABLE)
-          ) return false;
-        }
-
-        if (roles.length && !roles.includes(member.role)) return false;
+    // eslint-disable-next-line max-len
+    const isAvailable = (member) => member.weeklyCapacity - member.forecastedHours > AVAILABILITY_CUTOFF;
+    const availabilityFilter = (member) => {
+      if (availability.length === 0 || availability.length === 2) return true;
```

Comment:
We could have it check if `availability.length` is equal to the length of keys `availabilityOptions` has, but since adding another option 1) isn't something we foresee doing right now, and 2) would require a redesign anyway, I'm not letting this be a point of consideration.